src/libsync/clientsideencryption.cpp

```diff
@@ -1248,8 +1323,8 @@ void ClientSideEncryption::getPublicKeyFromServer(const AccountPtr &account)
                 _certificate = QSslCertificate(publicKey.toLocal8Bit(), QSsl::Pem);
                 _publicKey = _certificate.publicKey();
                 qCInfo(lcCse()) << publicKey;
-                qCInfo(lcCse()) << "Found Public key, requesting Private Key.";
-                getPrivateKeyFromServer(account);
+                qCInfo(lcCse()) << "Found Public key, requesting Server Public Key.";
+                getServerPublicKeyFromServer(account);
```

Comment:
@mgallien can you output `publicKey` in this log line? as a last part `<< " publicKey:" << publicKey`?